src/store/channels-list/utils.ts

```diff
@@ -4,7 +4,7 @@ import { denormalize } from './../channels/index';
 import getDeepProperty from 'lodash.get';
 import { select } from 'redux-saga/effects';
 import { currentUserSelector } from '../authentication/selectors';
-import { getUserHandle } from '../../components/messenger/list/utils/utils';
+import { getUserHandle } from '../../components/messenger/lib/utils';
```

Comment:
Redux stores shouldn't reference component code. Seems like we moved this to the wrong place. If it's not component specific code it should be in a more general lib folder.